samples/bluetooth/iso_connected_benchmark/README.rst

```diff
@@ -0,0 +1,111 @@
+.. _iso_broadcast_benchmark:
+
+Bluetooth: Throughput
+#####################
+
+The ISO Broadcast Benchmark sample tests the quality of connected ISO channels
+in terms for packet and sync loss.
```

Comment:
in terms "of"

Also unsure if "quality" is a good word to use here.  But as long as it is qualified with the "packet and sync loss", it should be OK.
An alternative might be something along the lines of :
"The ISO Connected Channels Benchmark sample measures and reports packet loss and sync loss in connected ISO channels."